bin/mne_x_plugins/resources/noisereduction/SPHARA/Current_SPHARA_EEG.txt

```diff
@@ -0,0 +1,64 @@
+0.12499999999999788 -0.006085511154443182 0.2176983177365003 -0.010056145367672714 -0.32415165965448234 -0.005090547851298449 0.04483967480029058 0.1571281947318599 0.35679416734870045 0.0068003066504849375 -0.13765207393075624 0.03224271050986667 0.14661720091421332 -0.3448752271003658 -0.010627108108393624 0.021273170318185215 0.0885846117104977 0.1950966384896189 -0.06461462947180473 -0.08570138365340735 -0.31176237395303674 0.08091031597623217 0.10362289886466775 -0.019641369830276104 0.1553524673189833 -0.1299367359475804 -0.045008034017812164 -0.24958031543527845 0.07990132827467011 -0.055243243212770334 0.02193100554478611 0.046959990203206695 0.0364303564622006 -0.0047321228113074 0.11778837093923328 0.23689908772089724 0.13245809099208622 0.027858514735383894 -0.04472028772793545 -0.15736793372741703 0.16962003143694165 -0.041405283260533644 0.17077775493717995 0.053578130777393554 -0.012404157614665796 -0.058947829003710546 -0.036698556162813326 0.041333317622101434 -0.06225237502599535 0.021742416187329666 0.10277489923300973 -0.07681928730327738 -0.06050342972067248 0.007633190069029475 0.03271313250841907 -0.047149854203795555 0.033160206383277144 0.00795151316346182 -0.07274185013113363 0.06323005010055338 0.03931777323108159 -0.010483235718714179 0.02932765908939847 0.022969114413344333
```

Comment:
This file is for a specific EEG setup - how about giving the file a self explaining name?